public/js/break_links.js

```diff
@@ -0,0 +1,5 @@
+var linksArray = document.querySelectorAll(".URL");
+
+linksArray.forEach(function(link){
+  link.innerHTML = link.innerHTML.substring(0, 27) + "...";
```

Comment:
E se ao invés desse [magic number](https://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad) `27` tivesse uma constante `const MAX_LINK_LENGTH = 27`? 😃